Source/WebCore/page/SecurityOriginData.h

```diff
@@ -25,15 +25,24 @@
 
 #pragma once
 
+#include "ProcessQualified.h"
+#include <wtf/ArgumentCoder.h>
 #include <wtf/Hasher.h>
+#include <wtf/Markable.h>
 #include <wtf/URL.h>
 
 namespace WebCore {
 
 class Frame;
 class SecurityOrigin;
 
+enum class ShouldSkipOpaqueSecurityOriginCheck : bool { No, Yes };
```

Comment:
I don't have a better proposal right now but this is ugly and so many call sites have to pass this parameter now..
It makes the new code look strictly worse (more verbose), which is unfortunate.

Reply:
Maybe instead of an enum parameter, we could use an overload? I feel that we could fine a name short enough that it would result in more concise code. E.g. `fromURL()` and `fromURLIgnoringOpaqueness()`? or `fromNonOpaqueURL()` ?